fuzz/fuzz_targets/chaos.rs

```diff
@@ -0,0 +1,78 @@
+#![no_main]
+use arbitrary::Arbitrary;
+use buddy_alloc::{BuddyAllocParam, FastAllocParam, NonThreadsafeAlloc};
+use libfuzzer_sys::fuzz_target;
+use std::alloc::{GlobalAlloc, Layout};
+use std::ptr::{addr_of, NonNull};
+
+#[derive(Debug, Arbitrary)]
+enum Action {
+    // Allocate a chunk with the size specified.
+    Alloc { size: u16, align_bit: u8 },
+    // Free the pointer at the index specified.
+    Free { index: u8 },
+}
+
+const FAST_HEAP_SIZE: usize = 32 * 1024; // 32 KB
+const HEAP_SIZE: usize = 1024 * 1024; // 1M
+const LEAF_SIZE: usize = 256;
+#[repr(align(64))]
+struct Heap<const S: usize>([u8; S]);
+static mut FAST_HEAP: Heap<FAST_HEAP_SIZE> = Heap([0u8; FAST_HEAP_SIZE]);
+static mut HEAP: Heap<HEAP_SIZE> = Heap([0u8; HEAP_SIZE]);
+
+fuzz_target!(|data: (u16, u32, u8, Vec<Action>)| {
+    let (fast_heap_size, heap_size, leaf_size, action_list) = data;
+    let fast_size = std::cmp::max(
+        64,
+        std::cmp::min(fast_heap_size as usize & 0xc0, FAST_HEAP_SIZE),
+    );
+    let heap_size = std::cmp::max(256, std::cmp::min(heap_size as usize & 0xc0, HEAP_SIZE));
```

Comment:
The 256 is too small. Suggest a more large number, like 16K.

Reply:
What is the reason for increasing its minimum value? Currently, the range of heap size is [256, 1024 * 1024], which will include the value of 16K.

Reply:
That's enough.